settings.py

```diff
@@ -1,19 +1,123 @@
+import os
+import sys
 import configparser
+import settings_dialog
+
+import PyQt5
+from PyQt5 import QtGui
+from PyQt5 import QtWidgets
+
+from PyQt5.QtGui import QFont
+
+from PyQt5.QtWidgets import QLabel
 """
- #LOAD SETTINGS
-    sets = loadConf()
-    sets['user']['default-recipe']
- #SAVE SETTINGS
-default
+ Settings with global variables
 """
-font=""
-def saveConf(defaultRecipe,fontName,fontSize,bookName):
-    config = configparser.ConfigParser()
-    config['user'] = {'default-recipe': str(defaultRecipe),'font-name': str(fontName),'font-size': int(fontSize), 'default-book-name': str(bookName)}
-    with open('configuration.ini', 'w') as configfile:
-        config.write(configfile)
-
-def loadConf():
-    config = configparser.ConfigParser()
-    config.read('configuration.ini')
-    return config
+
+pathsOfDocuments = []
+listPaths = []
+selectedRecipe = ""
+loadedRecipe = ""
+defaultRecipe = ""
+zipsFolder = ""
+tempFolder = ""
+localPath = ""
+sets = ""
+font = ""
+
+
+def isFolderExist(name):
```

Comment:
Tak jak jest na pewno nie może zostać, ja akurat lubię dawać prefixy to funkcji zwracających boola. Z drugiej strony ```dirExists``` jest bardzo naturalne, też skłaniam się do tego.